assets/js/common/TimeSeriesLineChart/TimeSeriesLineChart.jsx

```diff
@@ -0,0 +1,209 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Chart as ChartJS,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Tooltip,
+  TimeScale,
+  LogarithmicScale,
+  Legend,
+} from 'chart.js';
```

Comment:
What do you think about sorting the imports by alphabet? 
`import {
  Chart as ChartJS,
  Legend,
  LinearScale,
  LineElement,
  LogarithmicScale,
  PointElement,
  TimeScale,
  Tooltip,
} from 'chart.js';
`
It would increase readability a little :) ?

Reply:
done!